test/e2e/apimachinery/crd_publish_openapi.go

```diff
@@ -95,19 +95,19 @@ var _ = SIGDescribe("CustomResourcePublishOpenAPI [Privileged:ClusterAdmin]", fu
 
 		ginkgo.By("client-side validation (kubectl create and apply) rejects request with unknown properties when disallowed by the schema")
```

Comment:
rather than dropping the name of the field, check both forms of errors (client-side and server-side)

Reply:
done